docs/editors/overview.md

```diff
@@ -187,7 +202,7 @@ functionality.
     <td>Metals Extensions</td>
     <td align="center">✅</td>
     <td align="center">Status bar</td>
-    <td align="center">Status bar</td>
+    <td align="center">Status bar, Quick pick</td>
     <td align="center">Status bar</td>
```

Comment:
```suggestion
    <td align="center">Status bar, Input box</td>
```
Sublime plugin support Input Box now: https://github.com/scalameta/metals-sublime/blob/master/plugin.py#L114

Reply:
Thanks! I will add it! Should I add any of the documentation to sublime docs? The new project provider or parent code lenses?

Reply:
These two aren't yet supposed yet :/
But "new scala file" was just released: https://github.com/scalameta/metals-sublime/pull/26